packages/eslint-plugin-ts/rules/indent/indent.ts

```diff
@@ -181,6 +181,8 @@ export default createRule<RuleOptions, MessageIds>({
           rules['*:exit'](node)
       },
 
+      'PropertyDefinition': () => {},
```

Comment:
Could you please add test cases to test if it gets indented correctly if I use some line breaks inside a class field?
I'm worried that it won't work well because it overrides and erases the `PropertyDefinition` processing of the eslint-plugin-js `indent` rule.

Reply:
I have added few cases in 6ed5b09.

Reply:
The tests added were not what I intended. For example, break lines within the class field as below.

```js
class Foo {
    [
        x
    ] =
        [
            42
        ]
}
```

I don't think it will get indented well with the current PR changes.

Reply:
I see what you mean. Given this input

```ts
class Foo {
    [
      x
    ] = 
      [
        42
      ]
}
```

the output from this change is 

```ts
class Foo {
    [
    x
    ] = 
    [
        42
    ]
}
```

which is incorrect as `x` is missing 4 spaces, I'll try to delve deeper regarding this issue.

Reply:
After thinking through, I think it's better if we fixed it by handling `Decorator` in the JS plugin file.